GameData/masTerTorch/Localization/en-us.cfg

```diff
@@ -0,0 +1,104 @@
+	#KDex_MinmusInSpaceLowLesserFlats = The sensor indicates a moderate dust storm is in progress.
+	#KDex_MinmusInSpaceLowFlats = Composition of particles with diameter greater than 1my.
+	#KDex_MinmusInSpaceLowPoles = Finding a lack of coatings on the ionized dust particles.
+
+	#KDex_GillyInSpaceLow = Dust with silicate cores colliding in the back of the instrument.
+	#KDex_GillySrfLanded = These grains have a sooty mantle consisting of amorphous carbon.
+
+	#KDex_MohoInSpaceLow = Density is almost as high as the density of flying ash.
+	#KDex_MohoSrfLanded = Fine powder made up of glassy spheres from sub-micron to more than 100 microns in size. Specific gravity 1.9 to 2.4 and a bulk density of 0.8-1 ton per cubic meter.
+
+	#KDex_EveInSpaceLow = The dust is glass like. Strong impacts incoming.
+	#KDex_EveFlying = The heavy impacts and the high collision rate are quite a stress test.
+
+	#KDex_DunaInSpaceLow = Incoming Grains vary in size between a few tenths of a nanometer to a few microns.
+	#KDex_DunaSrfLanded = The dust is quite grainy and shows sand like collisions.
+
+	#KDex_IkeInSpaceLow = Dust particles are few and far between.
+	#KDex_IkeSrfLanded = Only a few dust particles are entering the sensor. We should have brought a shovel.
+
+	#KDex_DresInSpaceLow = The dust seems to be composed of carbonaceous, silicate, and metallic grains.
+	#KDex_DresSrfLanded = The dust is so dense that it absorbs and scatters sun light.
+
+	#KDex_JoolInSpaceLow = No dust or the grains are too small to be recognized by the instrument.
+
+	#KDex_LaytheInSpaceLow = The dust tries to from a shape. We observed the same behaviour near Kerbin.
+	#KDex_LaytheSrfLanded = A loose granular material. Probably it consists of calcium carbonate.
+	#KDex_LaytheSrfSplashed = No grains are entering the sensor.
+
+	#KDex_VallInSpaceLow = Dust tries to form a shape. Lower density than water.
+	#KDex_VallSrfLanded = The cold seems to trap the dust particles on the surface.
+
+	#KDex_TyloInSpaceLow = Avoid this one, it is just a lot of dust and drag and means nothing in the end.
+	#KDex_TyloSrfLanded = Due to the high gravity the dust clumps together to spheres.
+
+	#KDex_BopInSpaceLow = The dust is clustering to dense cores.
+	#KDex_BopSrfLanded = Too much dust in the sensor.
+
+	#KDex_PolInSpaceLow = Dusting this one would be a good start.
+	#KDex_PolSrfLanded = Weak impacts. Probably phosphoric or sulfuric origin.
+
+	#KDex_EelooInSpaceLow = Dust particles are few and far between. 
+	#KDex_EelooSrfLanded = The Dust consists of CO, silicon carbide, amorphous silicate and lots of ice.
+	
+	#KDEx_SunInSpaceLow = Dust particles are popping like popped corn; perhaps from being too close to Kerbal?!?!?
+	#KDEx_SunInSpaceHigh = You feel kind of small right now... You hope you know where you're going. Dust particles are popping like popped corn; perhaps from being too close to Kerbal?!?!?
```

Comment:
Copy-and-paste error or intentional to have the same text as SunInSpaceLow being part of this one?

Reply:
Am not very creative and sun in space would be -- well the same, only hotter. :D